src/index.js

```diff
@@ -20,6 +21,11 @@ module.exports = {
         return rslt;
       },
       verbose: true,
+      inputs: [{
```

Comment:
Eventually, these “question” objects should support different types (“text,” “number,” “date,” etc.). This is a basic `select` for now.